components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -47,37 +48,66 @@
       event_description,
       "",
     );
+    show_hosts = getPropertyValue(internalProps.show_hosts, show_hosts, "show");
   }
 
-  $: editableProperties = [
-    {
-      label: "Event Title",
-      value: event_title,
-    },
-    {
-      label: "Event Description",
-      value: event_description,
-    },
-  ];
+  $: manifestProperties = {
+    event_title,
+    event_description,
+    show_hosts,
+  };
   // #endregion mount and prop initialization
 
   function saveProperties() {
     console.log("Saving the following properties:");
-    console.log(editableProperties);
+    Object.entries(manifestProperties).forEach(([k, v]) => {
+      console.log(k, v);
+    });
```

Comment:
✂️ ❓

Reply:
Not just yet -- holding this in log until we actually write a manifest.save()